src/doc/src/reference/source-replacement.md

```diff
@@ -49,46 +16,64 @@ replacement Cargo can express:
   cache for crates.io itself.
 
 Cargo has a core assumption about source replacement that the source code is
-exactly the same from both sources. In our above example Cargo assumes that all
-of the crates coming from `my-awesome-source` are the exact same as the copies
-from `crates-io`. Note that this also means that `my-awesome-source` is not
-allowed to have crates which are not present in the `crates-io` source.
+exactly the same from both sources. Note that this also means that
+a replacement source is not allowed to have crates which are not present in the
+default crates.io source.
```

Comment:
I think I would word this last sentence slightly differently.  To me, it reads as-if *every* replacement must be the same as crates.io, which isn't true (there are now other registries).  Maybe something like the following?

```suggestion
original source.
```

Reply:
Agreed!